exporter/SynthesisFusionAddin/Synthesis.py

```diff
@@ -129,3 +130,14 @@ def register_ui() -> None:
     )
 
     gm.elements.append(apsButton)
+
+
+    websiteButton = HUI.HButton(
+        "Synthesis Website", 
+        work_panel,
+        Helper.check_solid_open,
```

Comment:
> When the addon starts, I get an error saying that 'UI.Helper' has no attribute 'check_solid_open' in `Synthesis.py:138`.

This was a merge conflict, left this to be changed in the HUI refactor.

```suggestion
        lambda *_: True,
```